css/index.css

```diff
@@ -124,10 +124,16 @@ body header nav p, a{
 		display: none;
```

Comment:
you have a chunk of code here with 3 media queries for 800px. really all of these can be in one media query of 800, you'll just stack the three elements you're changing in there (body header, body heaver nav, and .top-image)